src/pages/docs/contributing.md

```diff
@@ -1,13 +1,70 @@
 ---
-title: Contributing
+title: Contributing to Protocol
 order: 2
 ---
 
-We've done our best to make the barrier of entry to contributing code
-to Protocol as low as possible. You can even do a lot of stuff directly
-through GitHub without actually installing Protocol locally, but if you
-want to do any complex coding work you'll eventually need to set up a
-local development environment.
+Protocol, like everything else Mozilla makes, is open source. We welcome
+contributions from people like you. Yes, you, whoever you are reading this,
+can do something to make Protocol better and more useful, no matter your
+skill level or area of interest.
+
+### Reporting Bugs
+
+The easiest way to help is to report bugs. Let us know if something isn't
+doing what you think it should do and we can try to fix it.
+
+To report an error you’ve discovered in Protocol, [submit an issue on Github](https://github.com/mozilla/protocol/issues)
```

Comment:
Would it make more sense to point to the new issue screen? (https://github.com/mozilla/protocol/issues/new/choose)